### PR TITLE
3234 - Merging from main as we move to fix the webui

### PR DESCRIPTION
**Pull request by aronchick**
- Pulling in changes from main
- Lots of linting fixes - nothing was ever run
- Adding storyboard

### CODE REVIEW

**Review by simonwo — COMMENTED** [review]
This PR is HUGE – please can we limit PRs to just 1 thing at a time?

File: pkg/publicapi/apimodels/legacymodels/legacy.go [hunk 1]
```diff
@@ -21,6 +21,8 @@ type CancelResponse struct {
 	State *model.JobState `json:"state"`
 }
 
+type EmptyResponse struct{}
```
Comment (frrist) [inline pkg/publicapi/apimodels/legacymodels/legacy.go:24]: Unused

File: cmd/util/flags/configflags/api.go [hunk 2]
```diff
@@ -19,28 +19,6 @@ Ignored if BACALHAU_API_HOST environment variable is set.`,
...
-	are used by the server`,
-		EnvironmentVariables: []string{"BACALHAU_API_CACERT"},
-	},
-	{
-		FlagName:             "insecure",
-		DefaultValue:         Default.Node.ClientAPI.ClientTLS.Insecure,
-		ConfigPath:           types.NodeClientAPIClientTLSInsecure,
-		Description:          `Enables TLS but does not verify certificates`,
-		EnvironmentVariables: []string{"BACALHAU_API_INSECURE"},
-	},
```
Comment (frrist) [inline cmd/util/flags/configflags/api.go:43]: Don't remove these(?)

**Review by frrist — COMMENTED** [review]
I agree with Simon, its not possible (for me at least) to actually review 35K code changes.

**Comment by aronchick** [comment]
> This PR is HUGE – please can we limit PRs to just 1 thing at a time?

the problem is that linting, compilation, etc was never run on the repo - so there are tons and tons of basic changes from that perspective. 

the functionality of this repo is just to focus on those things - getting the project into a moderately decent shape moving forward. That said, I did also add storyboard - i'll split that into its own PR if you'd like.

**Comment by aronchick** [comment]
> the problem is that linting, compilation, etc was never run on the repo - so there are tons and tons of basic changes from that perspective.
> 
> the functionality of this repo is just to focus on those things - getting the project into a moderately decent shape moving forward. That said, I did also add storyboard - i'll split that into its own PR if you'd like.

the problem is that linting, compilation, etc was never run on the repo - so there are tons and tons of basic changes from that perspective.

the functionality of this repo is just to focus on those things - getting the project into a moderately decent shape moving forward. That said, I did also add storyboard - i'll split that into its own PR if you'd like.

**Pull request closed without merging**